cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -72,6 +76,32 @@ using v1::by_default;
 
 } // namespace method
 
+/// Represents result option flag
+/// Behaves like a regular :expr`enum`.
+struct result_option_id_t : public result_option_id {
```

Comment:
Why `_t` suffix is needed? We use this convention only for type aliases.

Reply:
Fixed